src/App.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
-import SignIn from './components/signin/SignIn';
+import SignIn from './components/signin/SignIn'
 import SignUp from './components/signup/SignUp';
 import { BrowserRouter as Router,Switch, Route } from 'react-router-dom';
+import Home from './components/home/Home';
 import Dashboard from './components/dashboard/Dashboard';
+import CardContainer from './components/cards/CardContainer';
 
 function App() {
   return (
     <Router>
 
       <Switch>
         <Route  exact path="/"
-        component={Dashboard}/>
+        component={Home}/>
         <Route path="/signin"
         component={SignIn}/>
         <Route path="/signup"
         component={SignUp}/>
+        <Route path="/dashboard"
+        component={Dashboard}/>
+        <Route path="/order"
```

Comment:
The order component should be called `Order`, to keep the convention in your routes.